Client/Ecosia/UI/Theme/EcosiaThemeColourPalette.swift

```diff
@@ -37,92 +41,90 @@ private class EcosiaDarkColourPalette: EcosiaLightColourPalette {
 
 private class EcosiaLightColourPalette: ThemeColourPalette {
     
-    private static var fallbackDefaultThemeManager: ThemeManager = {
-        DefaultThemeManager(sharedContainerIdentifier: AppInfo.sharedContainerIdentifier)
-    }()
+    let fallbackDefaultThemeManager: ThemeManager = DefaultThemeManager(sharedContainerIdentifier: AppInfo.sharedContainerIdentifier)
```

Comment:
It's a bit confusing to me that `fallbackDefaultThemeManager` belongs to `EcosiaLightColourPalette`. I guess that was already like this because `EcosiaDarkColourPalette` inherits the light one so not a blocker, but might be nice to move it into `ThemeColourPalette`.

Reply:
Thanks for checking this out @lucaschifino.
The `fallbackDefaultThemeManager` is something I'm using so to make sure `EcosiaTheme` with its colors' palette (light/dark) is used throughout the codebase instead of the Firefox's one.
I needed a fallback to make sure some of the references to Firefox's palette were used (following the light/dark alternatives) without having to copy each single single color we couldn't replace with our Ecosia ones as part for the `Ecosia(Light/Dark)ColourPalette`.
Moving it into `ThemeColourPalette` would mean modify `BrowserKit` which is something I tend to touch as less as possible when not needed. On top, it would create another point of failure when it comes to maintaining the codebase's colors